### PR TITLE
Added personal layout and userspace

### PR DESCRIPTION
**Pull request by FML128**
## Description

Added personal layout and userspace for massdrop/alt keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/massdrop/alt/keymaps/fml128/config.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#define COLOR HSV_GREEN // define solid color
```
Comment (drashna) [inline keyboards/massdrop/alt/keymaps/fml128/config.h:18]:
```suggestion
#pragma once

#define COLOR HSV_GREEN // define solid color
```

File: keyboards/massdrop/alt/keymaps/fml128/keymap.c [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+
+#ifdef RGB_MATRIX_ENABLE
+    static uint16_t idle_timer;
+    static uint8_t minute_counter;
+    static bool led_on;
```
Comment (drashna) [inline keyboards/massdrop/alt/keymaps/fml128/keymap.c:24]:
```suggestion
static uint16_t idle_timer;
static uint8_t minute_counter;
static bool led_on;
```

File: keyboards/massdrop/alt/keymaps/fml128/keymap.c [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    #ifdef RGB_MATRIX_ENABLE
+        rgblight_enable_noeeprom();
+        rgb_matrix_mode(RGB_MATRIX_SOLID_COLOR);
+        rgb_matrix_set_flags(LED_FLAG_ALL);
+        rgblight_sethsv(COLOR);
+
+        led_on = true;
+        idle_timer = timer_read();
+        minute_counter = 0;
+    #endif
```
Comment (drashna) [inline keyboards/massdrop/alt/keymaps/fml128/keymap.c:64]:
```suggestion
#ifdef RGB_MATRIX_ENABLE
    rgblight_enable_noeeprom();
    rgb_matrix_mode_noeeprom(RGB_MATRIX_SOLID_COLOR);
    rgb_matrix_set_flags(LED_FLAG_ALL);
    rgblight_sethsv_noeeprom(COLOR);

    led_on = true;
    idle_timer = timer_read();
    minute_counter = 0;
#endif
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: users/fml128/config.h [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+*/
+
+#undef MANUFACTURER
+#undef PRODUCT
+#undef DESCRIPTION
+
+
+#define MANUFACTURER        "FML128.ch"
+#define PRODUCT             "FMK128"
+#define DESCRIPTION         "FMK128"
```
Comment (drashna) [inline users/fml128/config.h:25]:
```suggestion
#define MANUFACTURER        FML128.ch
#define PRODUCT             FMK128
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: users/fml128/config.h (hunk 4 above)
Comment (FML128) [reply users/fml128/config.h]: This change resulted in an error. I have removed the description, but the " needs to be there.

**Review by drashna — APPROVED** [review]

File: keyboards/massdrop/alt/keymaps/fml128/keymap.c [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, \
+        _______, _______, _______,                            _______,                            KC_RCTL, _______, _______, _______, _______  \
+    ),
+    /*
+    [X] = LAYOUT(
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, \
+        _______, _______, _______,                            _______,                            _______, _______, _______, _______, _______  \
```
Comment (fauxpark) [inline keyboards/massdrop/alt/keymaps/fml128/keymap.c:49]:
```suggestion
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC, KC_DEL,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_HOME,
        _______, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,          KC_ENT,  KC_PGUP,
        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,          KC_UP,   KC_PGDN,
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, MO(1),   KC_LEFT, KC_DOWN, KC_RGHT
    ),
    [1] = LAYOUT_65_ansi_blocker(
        KC_GRV,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______,_______,_______, _______, _______, _______, _______, _______,
        RESET, _______,_______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
        _______, _______, _______,                            _______,                            KC_RCTL, _______, _______, _______, _______
    ),
    /*
    [X] = LAYOUT(
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______,
        _______, _______, _______,                            _______,                            _______, _______, _______, _______, _______
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**